core/src/apps/cardano/get_public_key.py

```
@@ -34,30 +46,31 @@ async def get_public_key(
         raise wire.ProcessError("Deriving public key failed")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)